subiquity/tests/api/test_api.py

```diff
@@ -735,6 +754,107 @@ async def test_win10(self):
             self.assertEqual(expected, sda1['os'])
 
 
+class TestRepartition(TestAPI):
```

Comment:
I'm not sure "repartition" is the right jargon here. TestPartitionTableEditing?

Reply:
Fixed, thanks!